README.md

```diff
@@ -28,6 +29,17 @@ impl FromRandom for MyStruct {
         }
     }
 }
+
+assert_eq!(0x55, 0b0101_0101);
+const SEED: [u8; 32] = [0x55; 32];
```

Comment:
:wrench: Not sure why this assert is here.
Since this is a doc example it seems like this will leave readers asking why it exists.

perhaps, if the intent is to communicate every other bit set:
```suggestion
const SEED: [u8; 32] = [0b0101_0101; 32];
```